sql/moz-fx-data-shared-prod/search_terms_derived/suggest_impression_sanitized_v2/query.sql

```diff
@@ -0,0 +1,83 @@
+/*
+
+This query is run in a standalone project and is scheduled separately from our
+normal Airflow infrastructure, but is codified here for discoverability and for
+making use of the testing harness here.
+
+Note that we need to fully qualify all table references since this runs from
+a separate project.
+
+*/
+WITH impressions AS (
+  SELECT
+    -- This should already be truncated to second level per CONSVC-1364
+    -- but we reapply truncation to be explicit about granularity.
+    TIMESTAMP_TRUNC(submission_timestamp, SECOND) AS submission_timestamp,
+    request_id,
+    search_query AS telemetry_query,
+    advertiser,
+    block_id,
+    context_id,
+    sample_id,
+    is_clicked,
+    locale,
+    metadata.geo.country,
+    metadata.geo.subdivision1 AS region,
+    normalized_os,
+    normalized_os_version,
+    release_channel,
+    position,
+    reporting_url,
+    scenario,
```

Comment:
could we also include `version` or `version` truncated to major version? might help with troubleshooting data in/consistency questions

Reply:
+1. Firefox version is one of the frequently used dimensions, let's include it here.

Reply:
Added `version` (truncated to major version) below.